src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const cost = days;
+
+  if (days >= 7) {
+    return cost * 40 - 50;
+  } else if (days >= 3) {
+    return cost * 40 - 20;
+  } else {
+    return cost * 40;
+  }
```

Comment:
move "magic" numbers to variables. I mean: if someone looks at your code, it's implicit, what is 40, 20, 50?
create some variables that will explain the meaning of the numbers.